pysb/tests/test_core.py

```diff
@@ -4,10 +4,31 @@
 import itertools
 
 
+@with_model
 def test_component_names_valid():
     for name in 'a', 'B', 'AbC', 'dEf', '_', '_7', '__a01b__999x_x___':
-        c = Component(name, _export=False)
+        c = Monomer(name, _export=False)
         eq_(c.name, name)
+        # Before the component is added, we should not be able to find it
+        assert not hasattr(model.comps, name)
+        # Add the element to a model and try to access it as attribute and item
+        model.add_component(c)
+        assert_equal(model.comps[name], c)
+        assert_equal(getattr(model.comps, name), c)
+
+
+@with_model
+def test_component_name_existing_attribute():
+    # _map is an attribute of ComponentSet, the superclass of Monomer
```

Comment:
ComponentSet is not the superclass of Monomer... These two comment lines can probably just be deleted, since the comment before the asserts explains the purpose.

Reply:
Removed the misleading comment as requested.